documentation/CONFIG.md

```diff
@@ -251,7 +250,7 @@ Also, these topic level configurations will be override the behavior of Connecto
 ``scylladb.offset.storage.table.enable``
 
   If true, Kafka consumer offsets will be stored in ScyllaDB table. If false, connector will skip writing offset 
-  information into ScyllaDB (this might imply duplicate writes into ScyllaDB when a task restarts).
```

Comment:
I think this still holds for counters and lists, no?

Reply:
This description suggested that: not using Scylla offsets implies duplicate writes. Therefore, I deleted the information about duplicate write. This does not mean that are none duplicate writes, only that "duplicate writes" issue is not documented here (because it is not affected by choosing Scylla vs Kafka offset storage).

Reply:
fair enough